corehq/apps/hqcase/tests/test_case_update_api.py

```diff
@@ -32,7 +33,7 @@ def setUpClass(cls):
         super().setUpClass()
         cls.domain_obj = create_domain(cls.domain)
         cls.web_user = WebUser.create(cls.domain, 'netflix', 'password', None, None)
-        cls.web_user.is_superuser = True  # in pre-release, this is superuser-only
+        cls.web_user.is_superuser = True
```

Comment:
should this be removed in favor of the edit data permission?

Reply:
:+1: just changed it